### ISSUE TITLE
Caller's Pigaches not in a drop pool

### ISSUE THREAD

**Issue by ghost** [LegionDark/Issues#1269]
**Date & Time**: 
12/9/2016 1406

**FFXI Client Version (use `/ver`)**: 
61103_1

**Server's Expected Client Version matches, yes/no? (use `$ecv`)**: 
No

**Character Name**: 
Immortality

**Nation**: 
Bastok

**Job(level)/Sub Job(level)**: 
99Thf/Dnc

**NPC or Monster or item Name**: 
Caller's Pigaches 

**Zone name**: 
N/a

**Coordinates (use `$where`)**: 
n/a

**ffxiah.com link (for items issues only)**: 
http://www.ffxiah.com/item/12102/callers-pigaches

**Multi-boxing? (multiple clients on same connection)**: 
Yes

**Steps To Reproduce / any other info**: 
Caller's Pigaches are not listed as having a drop from any mob like the rest of the AF3 feet.

**Comment by R3P0FFXI** [comment]
@JessLand86 ill be going through drop pools shortly once im done rescripting the scripted drops so some slots will open up

**Comment by TwIzTiD2469** [comment]
If I'm not mistaken aren't the pigaches also buyable from a moogle in port jeuno? pretty sure just need to be on summoner and talk to the moogle and buy them for 1m. theres 2 side by side, pretty sure its the one on the left. unless its been changed?

**Comment by JessLand86** [comment]
@TwIzTiD2469 They are buyable from the moogle, I noticed the feet did not drop from a mob like all the rest of the af3 feet do while creating the empy  armor page on the wiki. I figured it used to drop from something and got messed up somewhere along the way.

**Comment by TeoTwawki** [comment]
probably never was in any loot pools. when this server got its custom drops, base gear for empy armor was not among them. What does drop is what would drop on retail that has been in a stock darkrstar server base. On retail none of the empyrean feet are drops - they come from gold sturdy pyxis.

**Comment by TeoTwawki** [comment]
@R3P0FFXI huh..oddly I can't pull up any of the empyrean feet on the websites item tool, they should still be find-able just by being in item_basic even if they aren't dropped anyplace..wtf happened here..

**Comment by JessLand86** [comment]
@TeoTwawki odd, that's where I pulled all the drops from like 2 days ago when I created that page on the wiki.

**Comment by TeoTwawki** [comment]
Yeah I dunno whats up, they were there before and today when I went to look..none of the feet show.

**Comment by R3P0FFXI** [comment]
@JessLand86 all feet are being removed from drops and are already in shop as part of the drop rework currently ongoing

**Comment by TeoTwawki** [comment]
lol? reopening issues on dead tracker
